packages/internal/src/loaders/introspection.ts

```diff
@@ -56,12 +58,14 @@ export const NO_SUPPORTED_FEATURES: SupportedFeatures = {
   inputValueDeprecation: false,
   directiveArgumentsIsDeprecated: false,
   fieldArgumentsIsDeprecated: false,
+  typesIsOneOf: false,
 };
 
 /** Evaluates data from a {@link makeIntrospectSupportQuery} result to {@link SupportedFeatures} */
 export const toSupportedFeatures = (data: IntrospectSupportQueryData): SupportedFeatures => ({
   directiveIsRepeatable: _hasField(data.directive, 'isRepeatable'),
   specifiedByURL: _hasField(data.type, 'specifiedByURL'),
+  typesIsOneOf: _hasField(data.type, 'isOneOf'),
```

Comment:
nit: I would name this `inputOneOf` as per the deprecation name for the other feature for example (drop the "is", change to "input" from type)